server/index.js

```diff
@@ -95,6 +95,7 @@ global.LoginServer = require('../.server-dist/loginserver').LoginServer;
 global.Ladders = require('./ladders');
 
 global.Chat = require('./chat');
+global.__version = {head: global.Chat.version()};
```

Comment:
Can't ``global.Chat.version()`` here just be ``Chat.version()``?

Reply:
Triggers the linter IIRC

Reply:
I believe its usually `tsc` that complains abut this, but this file isnt typechecked so...